docs/tooling/packaging.md

```diff
@@ -0,0 +1,51 @@
+# Packaging PaSh
+
+`scripts/package` packages PaSh into various formats. It is a suitable
+replacement for deployments depending on `install.sh`,
+`setup-pash.sh`, `up.sh`, `distro-deps.sh`, and/or `pkg.sh`
+
+
+
+## Building Packages
+
+```
+  shell.sh
+```
+
+Enter a Docker container for building packages. Run without arguments
+to enter a REPL and print usage information for the `build` command.
+
+Since `shell.sh` is just a Bash, you can pass it commands.  For
+example, `./shell.sh -ic 'build 0.0.1 deb'` launches an interactive
```

Comment:
Maybe we should inform the user that he has to enable the experimental features for docker

Reply:
[6ff4e01](https://github.com/binpash/pash/pull/484/commits/6ff4e01f318350a83397b22643305fe90c8fadf0)